### PR TITLE
Fix issue #1266

### PR DESCRIPTION
**Pull request by diananeculai**
I created the add function in order to avoid duplicated code since addRecipient, addCC, addBCC and addReplyTo are almost the same. #1266

### CODE REVIEW

File: libraries/joomla/mail/mail.php [hunk 1]
```diff
@@ -179,16 +179,17 @@ public function setBody($content)
...
 	 *
-	 * @param   mixed  $recipient  Either a string or array of strings [email address(es)]
-	 * @param   mixed  $name       Either a string or array of strings [name(s)]
+	 * @param   mixed   $recipient  Either a string or array of strings [email address(es)]
+	 * @param   mixed   $name       Either a string or array of strings [name(s)]
+	 * @param   string  $method     The parent method's name.
 	 *
 	 * @return  JMail  Returns this object for chaining.
 	 *
 	 * @since   11.1
```
Comment (mbabker) [inline libraries/joomla/mail/mail.php:190]: For the new add method, please change the since tag to 12.2

**Comment by elinw** [comment]
Diana do you want to close 1266?

**Comment by chdemko** [comment]
All add\* methods should return $this for chaining as stated in the docblock

**Comment by diananeculai** [comment]
Done.

**Pull request merged**